src/shaders/disk_andrews2009_frag.glsl

```diff
@@ -47,14 +47,11 @@ void main(void)
 {
     // Compute the radius and height of this fragment
     float radius = sqrt(ModelPosition.x * ModelPosition.x + ModelPosition.y * ModelPosition.y);
-    float height = abs(ModelPosition.z);
+    float height = atan(abs(ModelPosition.z),radius);
```

Comment:
Why are you using atan2 here? The z value is the height above the midplane which is the height by definition.